README.md

```diff
@@ -153,6 +153,48 @@ usage is clear you can use `U`, `V` etc. for any following generic.
 If the usage is not obvious, you should use a more descriptive name. The same naming rules as for
 classes will apply then.
 
+**Do**
+```
+// use common abbreviations like T(ype), K(ey), V(alue), P(roperty) etc.
+type Partial<T> = { [P in keyof T]?: T[P]; };
+```
+
+```
+// add more semantic context by extending the type
+type ResponseData = Record<string, unknown>;
```

Comment:
perhaps add an example with an `unknown` type alias that is just there as a semantic label, see `SomeGenericValueLabel` at https://www.typescriptlang.org/play?#code/C4TwDgpgBAyg9gWwgcQgOwgJwJYGMBqAhgDYCuEAMoQEYTFQC8UpaA1mnAO5oBQPokWIhToseADwAVKBAAewdABMAzkKSoMOAiXJVaxAHyMoAbx5QoANx0QAXFEk8AvnwHQAGsfjrRW8cuAcNABzAyA

Reply:
Moved this to an issue